apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ClaimVerification/messages.ts

```diff
@@ -0,0 +1,13 @@
+import { defineMessage } from "react-intl";
+
+export const priorityTitle = defineMessage({
+  defaultMessage: "Priority status",
+  id: "66vzXk",
+  description: "Title for a candidates priority status",
+});
```

Comment:
Can reuse `IDNjBI`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/fc80f64b-cc9a-4069-888b-93d669471dfa)

Reply:
Thanks for point that out, I merged them in [combine messages for re-use](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10502/commits/9d76128b02d7ee705bea95ad5e3d14745b358b13)